msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Preapproval of RSC permissions
+description: In this article, learn more about to create, manage, and delete preapprovals of RSC enabled app in Microsoft Teams.
+ms.localizationpriority: medium
+ms.author: v-sdhakshina
+ms.topic: Conceptual
+ms.date: 08/29/2023
+---
+
+# Preapproval of RSC permissions
+
+Preapproval of resource-specific consent (RSC) permissions allows admins to approve RSC based on individual app in Microsoft Teams. Initially, admins had the ability to turn on or turn off RSC permission in the Teams Admin Center to either approve or block all RSC enabled apps.
+
+Admins can use preapproved RSC permissions to select specific apps for their users to install in Teams, without needing to approve or deny every app that requires these permissions.
+
+> [!NOTE]
+> Preapproval of RSC permissions is available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+
+## Set up PowerShell to manage preapproval of RSC permissions
+
+Preapproval of RSC permissions is managed through Microsoft Teams PowerShell. For more information, see [manage Teams with Teams PowerShell](/microsoftteams/teams-powershell-managing-teams).
+
+Use `Connect-MgGraph` to connect your PowerShell to Microsoft Graph and add the following permissions to create, manage, and delete preapproval policies:
+
+* `TeamworkAppSettings.ReadWrite.All`
+* `Policy.ReadWrite.Authorization`
+* `AppCatalog.Read.All`
+
+The following is an example of the PowerShell set up to manage preapproval of RSC permissions:
+
+```powershell
+connect-MgGraph -Scopes @('TeamworkAppSettings.ReadWrite.All', 'Policy.ReadWrite.Authorization', 'AppCatalog.Read.All','InformationProtectionPolicy.Read', 'ServicePrincipalEndpoint.ReadWrite.All', 'Policy.ReadWrite.PermissionGrant', 'Policy.ReadWrite.ApplicationConfiguration', 'Application.ReadWrite.All')
+```
+
+## Enable the preapproval of RSC permissions
+
+To enable preapproval for RSC permissions, change your RSC settings with the `Set-MgBetaChatRscConfiguration` and `Set-MgBetaTeamRscConfiguration` cmdlets to allow preapproved apps. The following table list the available configurations for RSC settings:
+
+|Configuration| Description|
+|---|---|
+|`ApprovedForAllApps`|You can allow users in your tenant to access any RSC enabled app. After you enable this configuration, there's no need for preapproval of RSC permissions, as all RSC enabled apps are automatically approved for use.|
```

Comment:
ManagedByMicrosoft state needs to be added?

Reply:
Added as suggested.